internal/cli/quickstarts.go

```diff
@@ -31,15 +40,21 @@ var (
 		}
 		return
 	}()
-)
 
-// QuickStart app types and defaults
-const (
-	qsNative    = "native"
-	qsSpa       = "spa"
-	qsWebApp    = "webapp"
-	qsBackend   = "backend"
-	_defaultURL = "http://localhost:3000"
+	clientID = Flag{
```

Comment:
Can you please prefix it with `qs` (`qsClientID`), until we move it to its own package?